src/components/Menu/MenuModalWrapper.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Children } from 'react';
+import styled, { useTheme } from 'styled-components';
+
+// Icons
+import { IoChevronBackCircleOutline } from 'react-icons/io5';
+
+// Utils
+import { Theme } from '../../utils/theme';
+
+const MenuModalWrapper = ({
+  children,
+  title,
+  onBackButtonClick,
+}: {
+  children: React.ReactNode;
+  title: string;
+  onBackButtonClick?: () => void;
+}) => {
```

Comment:
Since you have multiple props, probably better to have an interface.
```suggestion
interface IMenuModalWrapper {
  children: React.ReactNode;
  title: string;
  onBackButtonClick?: () => void;
};

const MenuModalWrapper = ({
  children,
  title,
  onBackButtonClick,
}: IMenuModalWrapper) => {
```